gips/datahandler/logger.py

```diff
@@ -33,8 +33,10 @@ def __new__ (cls):
     def __init__(self):
         self.extra = {}
         try:
+            # TODO this can't work for RQ
             self.extra['jobid'] = os.environ['PBS_JOBID']
         except:
+            # TODO is this correct?
```

Comment:
I thought this was reasonable. Some things that log are not queued jobs -- so just use the script name (scheduler, geokitd). Idea being that the log needs to report something about what is doing the logging. 

But the try-except will probably not work without modification. It currently depends on failing on the lack of env var. Should be rolled into get_job_name and made clever.